packages/dd-trace/test/plugins/versions/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const latests = require('./package.json').dependencies
+
+/**
+ * @param {string} name
+ * @param {string} range
+ */
+function getCappedRange (name, range) {
+  const alreadyCapped = range.split('||').every(r => {
+    return r.includes('-') || r.includes('<') || /^=?\d+\.\d+\.\d+$/.test(r)
```

Comment:
Would that work with ranges like `4` or `4.1` ?

Reply:
Ok I fixed the logic to handle everything that is not an exact range.